projects/aca-playwright-shared/src/page-objects/components/adf-info-drawer.component.ts

```diff
@@ -36,4 +36,32 @@ export class AdfInfoDrawerComponent extends BaseComponent {
   public getIdField = (labelText: string) => this.getChild('[data-automation-id="library-id-properties-wrapper"]', { hasText: labelText });
   public getVisibilityField = (labelText: string) => this.getChild('[data-automation-id="library-visibility-properties-wrapper"]', { hasText: labelText });
   public getDescriptionField = this.getChild('[data-automation-id="library-description-properties-wrapper"] textarea');
+  public propertiesTab = this.getChild('.mat-tab-label-content').nth(0);
+  public commentsTab = this.getChild('.mat-tab-label-content').nth(1);
+  public commentInputField = this.getChild('mat-form-field');
+  public commentsHeader = this.getChild('#comment-header');
+  public addCommentButton = this.getChild('[data-automation-id="comments-input-add"]');
+  public commentsList = this.getChild('.adf-comment-list-item');
+  public commentUsername = this.getChild('.mat-line.adf-comment-user-name');
+  public commentTextContent = this.getChild('.mat-line.adf-comment-message');
+  public commentTimestamp = this.getChild('.mat-line.adf-comment-message-time');
+  public commentProfileIcon = this.getChild('.adf-comment-user-icon.ng-star-inserted');
```

Comment:
I think you don't need `.ng-star-inserted` as this is internal Angular class

Reply:
You're right. Updated, will be fixed with the next push.